json.c

```diff
@@ -290,6 +290,29 @@ static struct json_object *path_info(int fd, uint32_t blob_id)
 	return obj;
 }
 
+static struct json_object *fb_info(int fd, uint32_t id)
+{
+	if (id == 0) {
+		return NULL;
+	}
```

Comment:
id is already checked before calling this function. Remove either this or the check before this function is called.